frontend/src/components/widgets/CardBadges.jsx

```diff
@@ -7,8 +7,8 @@ import SentimentSatisfiedIcon from "@material-ui/icons/SentimentSatisfied";
 import SentimentDissatisfiedIcon from "@material-ui/icons/SentimentDissatisfied";
 import CallMergeIcon from "@material-ui/icons/CallMerge";
 import MoodBadIcon from "@material-ui/icons/MoodBad";
-import { Chip, Tooltip } from "@material-ui/core";
-
+import { Tooltip } from "@material-ui/core";
+import { Chip } from "@material-ui/core";
```

Comment:
It should be like this:
```jsx
import Tooltip from "@material-ui/core/Tooltip";
import Chip from "@material-ui/core/Chip";
```
Otherwise you're still pulling in the whole library for just one or two components